.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,33 @@
-Please read the [contribution guidelines](https://github.com/NatronGitHub/Natron/blob/master/CONTRIBUTING.md).
+Thanks for submitting a pull request! Please provide enough information so that others can review your pull request. Additionally, make sure you've done all of these things:
 
-## Description
+- [ ] I've followed the [contributing guidelines](https://github.com/NatronGitHub/Natron/blob/RB-2.4/CODE_OF_CONDUCT.md) to the best of my understanding
```

Comment:
More links that will break in the future :(

Reply:
Leave it for now? Until the new website goes up (where presumably we will put the contributing guidelines and code of conduct) we have no real other option.